src/test/java/ua/com/sportfood/services/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package ua.com.sportfood.services;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import ua.com.sportfood.dao.CustomerDAO;
+import ua.com.sportfood.formRegistration.RegistrationForm;
+import ua.com.sportfood.models.Customer;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class RegistrationServiceImplTest {
```

Comment:
Remove public modifier due to sonar rule:
https://rules.sonarsource.com/java/RSPEC-5786

"JUnit5 test classes and methods should have default package visibility"

Reply:
Done